src/libm-tester/tester.c

```diff
@@ -47,6 +47,7 @@ void stop(char *mes) {
 
 int ptoc[2], ctop[2];
 int pid;
+FILE *fpctop;
```

Comment:
Does it have to be global? Can't it be private inside of main?

Reply:
fpctop is reference by many functions.